src/e2e/actor/createItemMacro.test.js

```diff
@@ -0,0 +1,72 @@
+import { createOseMacro } from '../../module/macros';
+import { trashChat, waitForInput } from '../testUtils';
+
+
+export const key = 'ose.actor.macro';
+export const options = {
+    displayName: 'The Character Sheet: Item Macros'
+};
+
+export default ({ before, beforeEach, after, describe, it, expect, assert, ...context }) => {
+    const testCharacterName = 'Quench Test Character';
+    const prepareActor = async (data) => {
+        await trashChat();
+        await trashActor();
+
+        return Actor.create({
+            ...data,
+            name: testCharacterName,
+            type: 'character'
+        });
+    };
+    const testActor = () => game.actors.getName(testCharacterName);
+    const trashActor = () => testActor()?.delete();
+    const trashMacro = () => (game.macros.size)
+        ? game.macros.documentClass.deleteDocuments([], { deleteAll: true })
```

Comment:
we haven't made a test this destructive before. I used to assume non-coders could run tests with Quench themselves, without having to warn them not to use their home worlds.

I'm not sure this needs to be changed but if we can find the testMacro and delete just that, it'd be ideal

Reply:
Oh yes, you are 100% right. I'll correct this!